content/docs/command-reference/import-url.md

```diff
@@ -147,7 +147,8 @@ produces a regular stage in `dvc.yaml`.
   is useful if you need track changes in remote data without using local storage
   space (yet). The data can be downloaded later using `dvc pull`, but this will
   fail if the `url` no longer matches the hash values. File hashes can be
-  updated using `dvc update --no-download`.
+  updated using
+  [`dvc update --no-download`](/doc/command-reference/update#--no-download).
```

Comment:
This should auto-link correctly 🙂 

```suggestion
  updated using `dvc update --no-download`.
```

Reply:
Yeah it already links well, see https://dvc.org/doc/command-reference/import#--no-download.